Ruby/book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
```

Comment:
- [ ] It would be better if we set up the relationship between the classroom and the student, for example;  student belongs to the classroom kind of relations. Also while setting the classroom for a student it also adds it to the classrooms' students. Kindly address these :+1: Here is a [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships) that can help us understand the relationships